packages/libs/react-ui/src/styles/vars.css.ts

```diff
@@ -90,126 +91,126 @@ export const vars = createGlobalTheme(':root', {
     $3xl: 'var(--spacing-3xl)',
   },
   colors: {
-    $white: '#FFFFFF',
-    $black: '#000000',
-
-    $foreground: '#050505',
-    $background: '#FAFAFA',
-
-    $neutral1: '#FAFAFA',
-    $neutral2: '#F0F0F0',
-    $neutral3: '#9EA1A6',
-    $neutral4: '#474F52',
-    $neutral5: '#1D1D1F',
-    $neutral6: '#050505',
-
-    $primaryAccent: '#2997FF',
-    $primarySurface: '#C2E1FF',
-    $primaryContrast: '#00498F',
-    $primaryHighContrast: '#002F5C',
-
-    $secondaryAccent: '#ED098F',
-    $secondarySurface: '#FB93D0',
-    $secondaryContrast: '#A30662',
-    $secondaryHighContrast: '#580335',
-
-    $tertiaryAccent: '#9D00FF',
-    $tertiarySurface: '#C870FF',
-    $tertiaryContrast: '#6400A3',
-    $tertiaryHighContrast: '#320052',
-
-    $borderDefault: '#E4E6E7',
-    $borderSubtle: '#D8D9DB',
-    $borderContrast: '#9EA1A6',
-
-    $layoutSurfaceDefault: '#FAFAFA',
-    $layoutSurfaceSubtle: '#F0F0F0',
-    $layoutSurfaceContrast: '#FFFFFF',
-    $layoutSurfaceOverlay: '#757575',
-
-    $infoAccent: '#2997FF',
-    $infoSurface: '#DBEDFF',
-    $infoContrast: '#00498F',
-    $infoHighContrast: '#002F5C',
-
-    $positiveAccent: '#5EEA15',
-    $positiveSurface: '#E5FFD8',
-    $positiveContrast: '#1F6100',
-    $positiveHighContrast: '#113300',
-
-    $warningAccent: '#FF9900',
-    $warningSurface: '#FFE7C2',
-    $warningContrast: '#704300',
-    $warningHighContrast: '#3D2500',
-
-    $negativeAccent: '#FF3338',
-    $negativeSurface: '#FFE7E5',
-    $negativeContrast: '#75000B',
-    $negativeHighContrast: '#410006',
-
-    $disabledContrast: '#C5C7CA',
+    ...colorPallet,
+    ...gradients,
+
+    $background: colorPallet.$gray10,
+    $foreground: colorPallet.$gray100,
+
+    $neutral1: colorPallet.$gray10,
+    $neutral2: colorPallet.$gray20,
+    $neutral3: colorPallet.$gray40,
+    $neutral4: colorPallet.$gray60,
+    $neutral5: colorPallet.$gray90,
+    $neutral6: colorPallet.$gray100,
+
+    $primarySurface: colorPallet.$blue40,
+    $primaryContrast: colorPallet.$blue80,
+    $primaryHighContrast: colorPallet.$blue100,
+    $primaryAccent: colorPallet.$blue60,
+
+    $secondarySurface: colorPallet.$pink40,
+    $secondaryContrast: colorPallet.$pink80,
+    $secondaryHighContrast: colorPallet.$pink100,
+    $secondaryAccent: colorPallet.$pink60,
+
+    $tertiarySurface: colorPallet.$purple40,
+    $tertiaryContrast: colorPallet.$purple80,
+    $tertiaryHighContrast: colorPallet.$purple100,
+    $tertiaryAccent: colorPallet.$purple60,
+
+    $borderDefault: hexToRgba(colorPallet.$gray50, 0.2),
+    $borderSubtle: hexToRgba(colorPallet.$gray40, 0.4),
```

Comment:
Are you sure these should be transparent? Normally I would not expect borders to be transparent

Reply:
yep following the figma they should have
https://www.figma.com/file/cNQkFOjrqO3PAYv7TSIhpB/Foundation?node-id=122%3A2631&mode=dev